package.json

```diff
@@ -520,6 +521,11 @@
         "command": "gistpad.uploadFileToGist",
         "title": "Upload File(s)",
         "category": "GistPad"
+      },
+      {
+        "command": "gistpad.viewForkedGist",
+        "title": "View Forked Gist",
```

Comment:
Should we call this `View Forks`? Since it's displaying a list of the forks for the selected gist.

Reply:
Good idea! I will change this.